docs/sources/release-notes/v2.4.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Grafana Mimir version 2.4 release notes"
+menuTitle: "V2.4 release notes"
+description: "Release notes for Grafana Mimir version 2.4"
+weight: 300
+---
+
+# Grafana Mimir version 2.4 release notes
+
+Grafana Labs is excited to announce version 2.4 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+**Note**: If you are upgrading from Grafana Mimir 2.3, review the [list of important changes](#important-changes) that follow.
+
+## Features and enhancements
+
+- **Query-scheduler ring-based service discovery**:
+  The [query-scheduler]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md" >}}) is an optional, stateless component that retains a queue of queries to execute, and distributes the workload to available queriers. The use the query-scheduler, query-frontends and queriers are required to discover the addresses of the query-scheduler instances.
+
+  Other than the DNS-based service discovery, Mimir 2.4 introduces the ring-based service discovery for the query-scheduler. When enabled, the query-schedulers join their own hash ring (similar to other Mimir components), and the query-frontends and queriers discover query-scheduler instances via the ring.
+
+  The ring-based service discovery makes easier to setup the query-scheduler in environments where you can't easily define a DNS entry resolving to the running query-scheduler instances. Refer to [query-scheduler configuration]({{< relref "../operators-guide/architecture/components/query-scheduler/index.md#configuration" >}}) for more information.
+
+- **New API endpoint exposing per-tenant limits**:
+  Mimir 2.4 introduces a new API endpoint, available on all Mimir components that load the runtime configuration, exposing the limits for the authenticated tenant. This new API endpoint can be used when developing custom integrations with Mimir that require looking up the actual limits applied on a given tenant. Refer to [Get tenant limits]({{< relref "../operators-guide/reference-http-api/index.md#get-tenant-limits" >}}) for more information.
+
+- **New TLS configuration options**:
+  Mimir 2.4 introduces new options to configure the accepted TLS cipher suites and minimum version for the HTTP and gRPC clients used between Mimir components, or by Mimir to communicate to external services (e.g. Consul or etcd).
+
+  These new configuration options can be used to override the default TLS settings and meet your security policies. Refer to [Securing Grafana Mimir communications with TLS]({{< relref "../operators-guide/secure/securing-communications-with-tls.md" >}}) for more information.
```

Comment:
```suggestion
  You can use these new configuration options to override the default TLS settings and meet your security policy requirements. For more information, refer to [Securing Grafana Mimir communications with TLS]({{< relref "../operators-guide/secure/securing-communications-with-tls.md" >}}).
```
Do you mean "... and meet your security ..." or "... to meet your security ..."?